streams/streams-scala/src/test/resources/log4j2.properties

```diff
@@ -16,19 +16,26 @@
 #   See the License for the specific language governing permissions and
 #   limitations under the License.
 
-# Set root logger level to DEBUG and its only appender to A1.
-log4j.rootLogger=INFO, R
+name=StreamsScalaTestConfig
+appenders=console,rolling
 
-# A1 is set to be a ConsoleAppender.
-log4j.appender.A1=org.apache.log4j.ConsoleAppender
+appender.console.type=Console
+appender.console.name=A1
 
-log4j.appender.R=org.apache.log4j.RollingFileAppender
-log4j.appender.R.File=logs/kafka-streams-scala.log
+appender.rolling.type=RollingFile
+appender.rolling.name=R
+appender.rolling.fileName=logs/kafka-streams-scala.log
+appender.rolling.filePattern=logs/kafka-streams-scala.log.%i
+appender.rolling.layout.type=PatternLayout
+appender.rolling.layout.pattern=%-4r [%t] %-5p %c %x - %m%n
 
-log4j.appender.R.MaxFileSize=100KB
-# Keep one backup file
-log4j.appender.R.MaxBackupIndex=1
+appender.rolling.policies.type=Policies
+appender.rolling.policies.size.type=SizeBasedTriggeringPolicy
+appender.rolling.policies.size.size=100KB
 
-# A1 uses PatternLayout.
-log4j.appender.R.layout=org.apache.log4j.PatternLayout
-log4j.appender.R.layout.ConversionPattern=%-4r [%t] %-5p %c %x - %m%n
+appender.rolling.strategy.type=DefaultRolloverStrategy
+appender.rolling.strategy.max=1
+
+rootLogger.level=INFO
+rootLogger.appenderRefs=R
+rootLogger.appenderRef.R.ref=R
```

Comment:
Yes, the name `R` might be confusing. My intention was to purely transform from log4j1 to log4j2 without making any further unnecessary changes. However, it can definitely be renamed if needed.